openfisca_dummy_country/parameters/param_more.xml

```diff
@@ -1,35 +1,42 @@
 <NODE code="crds" description="Contribution au remboursement de la dette sociale (CRDS)">
   <NODE code="activite" description="Revenus d'activité">
     <CODE code="taux" description="taux de la CRDS" format="percent">
-      <VALUE deb="1998-01-01" fin="2014-12-31" valeur=".005" />
+      <END deb="2015-01-01" />
+      <VALUE deb="1998-01-01" valeur=".005" />
     </CODE>
     <BAREME code="abattement" description="Abattement sur les revenus d'activité, du chômage et des préretraites">
       <TRANCHE code="tranche0">
         <TAUX>
-          <VALUE deb="1998-01-01" fin="2004-12-31" valeur=".05" />
-          <VALUE deb="2005-01-01" fin="2011-12-31" valeur=".03" />
-          <VALUE deb="2012-01-01" fin="2014-12-31" valeur=".0175" />
+          <END deb="2015-01-01" />
```

Comment:
Good point. I'm sure we could find a way to avoid this repetition, but it's ok for now, we don't want to dig too much into this.